CMakeLists.txt

```
@@ -169,8 +173,10 @@ adios_option(AWSSDK     "Enable support for S3 compatible storage using AWS SDK'
 include(${PROJECT_SOURCE_DIR}/cmake/DetectOptions.cmake)
 
 if(ADIOS2_HAVE_CUDA)
```

```suggestion
if(ADIOS2_HAVE_CUDA OR ADIOS2_HAVE_Kokkos_CUDA)
```

This should fix the warnings

---

Agree with this one, will push the change